ghc-prof-flamegraph.cabal

```diff
@@ -6,6 +6,7 @@ license-file:        LICENSE
 author:              Francesco Mazzoli
 maintainer:          francesco@fpcomplete.com
 copyright:           (c) 2015 FP Complete Corporation
+                     (c) 2016 Seagate Technology Limited
```

Comment:
I'm not sure this copyright notice is warranted, how does it usually work in these cases? In my experience, PRs do not result in a new copyright notice in the .cabal file.